evalml/demos/diabetes.py

```diff
@@ -13,12 +14,17 @@ def load_diabetes():
         (pd.Dataframe, pd.Series): X and y
     """
     filename = (
-        "https://api.featurelabs.com/datasets/diabetes.csv?library=evalml&version="
+        "https://api.featurelabs.com/datasets/diabetes-2022-06-27.csv?library=evalml&version="
         + evalml.__version__
     )
     X, y = load_data(filename, index=None, target="target")
     y.name = None
-
+    X = X.astype(float)
+    y = y.astype(float)
+    numpy_of_X = X.to_numpy()
+    numpy_of_X = scale(numpy_of_X, copy=False)
```

Comment:
You can simplify these two lines to just `X_np = scale(X)`!

(I'd personally rename `numpy_of_X` to `X_np` for conciseness' sake)